test/testdata/demo_long.py

```diff
@@ -39,14 +39,14 @@
 
 FOO_CONSTANT: int = 42
 """
-A happy constant. ✨  
+A happy constant. ✨
```

Comment:
Sorry, it seems that my VS Code trimmed some trailing whitespaces. Please let me know if you want them back.